test/test_other.py

```diff
@@ -2875,6 +2875,17 @@ def test_jspi_wildcard(self):
 
     self.do_runf(test_file('other/test_jspi_wildcard.c'), 'done')
 
+  def test_jspi_addfunction(self):
```

Comment:
Add underscore `test_jspi_add_function` and the filename too.

Reply:
done